ReleaseNotes.md

```diff
@@ -1,5 +1,55 @@
 # Release Notes
 
+## New in Release 2.5.0
+Compression-aware training:
+- New Features:
+  - Added Patcher class from OTX, which is used for patching the models that can not be wrapped properly.
+  - (PyTorch, Tensorflow) Added `strip` that convert compressed model to inference in backend format without NNCF specific operations (with int4 support).
+- Fixes:
+  - Fixed traced tensors to implement the YOLOv8 from Ultralytics.
+  - Fixed statistics computation for pruned layers.
+- Improvements:
+  - Extension of attributes (`transpose/permute/getitem`) for pruning node selector.
+  - NNCFNetwork was refactored from a wrapper-approach to a mixin-like approach.
+  - Added configuration for Accuracy Aware (pruning + quantization) for Resnet18 on CIFAR10.
```

Comment:
(PyTorch) Added config file for ResNet18 accuracy-aware pruning + quantization on CIFAR10